ChangeLog.txt

```diff
@@ -108,6 +112,8 @@ MINOR:
 - Made unreformed pagans twice as likely to start independence factions
 - Added Glycerius' assasination attempt in the Nepos startdate
 - Moved Odoacer to Mantua and turned the initial war against Romulus Augustus into a rebellion
+- Powerful Melchisidechians can now restore the Bronze Age Kingdom of Salem and crown themselves the spiritual successor to the Priest-King Melchizidek
+- Tweaked gods and religious flavor for Tzakidite Jews due to their Kabbalism
```

Comment:
I only include the changes that are mentioned. :wink: And regardless, the purpose of the changelog is for releases.